client/stylesheets/_toCat.scss

```diff
@@ -5445,4 +5445,33 @@ On click the images will be displayed at normal size to complete the effect
 
 .listingPayCash i {
 	margin-right: 5px;
-}
\ No newline at end of file
+}
+
+
+/** Amount Profile **/
+
+.profileUserAmount ul {
+    margin: 8px 0 !important;
+}
+
```

Comment:
Merge rule `.profileUserAmount ul li` with rule on line 5453<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `profileUserAmount` should be written in lowercase with hyphens